controllers/drplacementcontrol.go

```diff
@@ -1549,7 +1549,7 @@ func (d *DRPCInstance) ensureClusterDataRestored(homeCluster string) (*rmn.Volum
 }
 
 func (d *DRPCInstance) EnsureCleanup(clusterToSkip string) error {
-	d.log.Info("ensuring cleanup on secondaries")
+	d.log.Info("Ensure cleanup on secondaries")
```

Comment:
I'm not sure this is better, but it is not part of logging level changes. It will be better to do logging text improvements in a separate commit.

Reply:
Restored